components/brave_vpn/common/brave_vpn_utils.cc

```diff
@@ -95,6 +96,28 @@ std::string GetManageUrl(const std::string& env) {
   return brave_vpn::kManageUrlProd;
 }
 
+// On desktop, the environment is tied to SKUs because you would purchase it
+// from `account.brave.com` (or similar, based on env). The credentials for VPN
+// will always be in the same environment as the SKU environment.
+//
+// When the vendor receives a credential from us during auth, it also includes
+// the environment. The vendor then can do a lookup using Payment Service.
+std::string GetBraveVPNPaymentsEnv(const std::string& env) {
+  if (env == skus::kEnvProduction)
+    return "";
+  // Use same value.
+  if (env == skus::kEnvStaging || env == skus::kEnvDevelopment)
+    return env;
+
+  NOTREACHED();
+
+#if defined(OFFICIAL_BUILD)
```

Comment:
was this moved from somewhere or new code? Why is there any code after NOTREACHED?

Reply:
I understand it has to return something from the method for the compiler, but why would it depend on OFFICIAL_BUILD when it should never be reached in the first place?

Reply:
https://github.com/brave/brave-core/blob/master/components/brave_vpn/brave_vpn_service_helper.cc#L59 
maybe @simonhong can help to figure out